src/main/scala/org/broadinstitute/dsde/firecloud/service/EntityService.scala

```diff
@@ -71,16 +69,6 @@ trait EntityService extends HttpService with PerRequestCreator with FireCloudDir
                     passthrough(requestCompression = true, entityTypeUrl, HttpMethods.GET)
                   }
                 } ~
-                  parameters('attributeNames.?) { attributeNamesString =>
-                    path("tsv") {
-                      requireUserInfo() { userInfo => requestContext =>
-                        val filename = entityType + ".txt"
-                        val attributeNames = attributeNamesString.map(_.split(",").toIndexedSeq)
-                        perRequest(requestContext, ExportEntitiesByTypeActor.props(exportEntitiesByTypeConstructor, userInfo),
-                          ExportEntitiesByTypeActor.ExportEntities(workspaceNamespace, workspaceName, filename, entityType, attributeNames))
-                      }
-                    }
-                  } ~
```

Comment:
Specifically pulled the tsv download out of this service due to interference from passthrough route problems. Ticket for that is here: https://broadinstitute.atlassian.net/browse/GAWB-1303